packages/flutter/lib/src/widgets/animated_scroll_view.dart

```diff
@@ -131,10 +131,18 @@ class AnimatedList extends _AnimatedScrollView {
 /// animation is passed to [AnimatedList.itemBuilder] whenever the item's widget
 /// is needed.
 ///
+/// When multiple items are inserted with [insertAllItems] an animation begins running.
+/// The animation is passed to [AnimatedList.itemBuilder] whenever the item's widget
+/// is needed.
+///
 /// When an item is removed with [removeItem] its animation is reversed.
 /// The removed item's animation is passed to the [removeItem] builder
 /// parameter.
 ///
+/// When all items are removed with [removeAllItems] its animation is reversed.
```

Comment:
These paragraphs start to read a little redundantly now. Perhaps the original 2 paragraphs (for insert and remove here) can be rewritten to be more concise, and refer to both the individual and all methods?